### ISSUE TITLE
Order of learning rate augmentation

### ISSUE THREAD

**Issue by AlexYangLi** [bckenstler/CLR#15]
> Note that the clr callback updates the learning rate prior to any further learning rate adjustments as called for in a given optimizer.  

Hi, @bckenstler , excellent work! I am still confused about what you said about the "order of learning rate augmentation". If the clr callback is added and sets the learning rate after each batch training ends, will a given optimizer (e.g. adam) still adjust the learning rate that the clr just set for updating weights. Thanks!
